src/Abp/Application/Features/Feature.cs

```diff
@@ -63,23 +64,20 @@ public class Feature
 
         /// <summary>
         /// Default value of the feature.
-        /// This value is used if feature's value is not defined for current edition or tenant.
+        /// This value is used if the feature's value is not defined for a current edition or tenant.
```

Comment:
- "Default value of `this` feature."
- "`this` feature's value is not defined for `the` current edition or tenant."